test/helpers.ts

```diff
@@ -89,3 +93,11 @@ export class CodecRegistry {
     return this._codecs.get(key)
   }
 }
+
+export const newLocalDockerClient = (): Promise<Client> =>
+  Client.create(newWallet(), {
+    bootstrapAddrs: [LOCAL_DOCKER_MULTIADDR],
+  })
+
+export const newDevClient = (): Promise<Client> =>
+  Client.create(newWallet(), { env: 'dev' })
```

Comment:
Where does `dev` translate into a node list?

Reply:
Oh ya, we have our own fleets file. https://github.com/xmtp-labs/xmtp-nodes-service